### ISSUE TITLE
Processing error

### ISSUE THREAD

**Issue by boxerone** [OpenDroneMap/WebODM#51]
Hi there
I am not sure if this error is related to webODM or the processig node

My setup is two different AWS instance (t2.medium) one running WebODM and the second one running Node-ODM, both installed with Dockers

```
Killed
[ERROR]   quitting cause: 
	/code/SuperBuild/install/bin/pmvs2 /var/www/data/813e7f99-16a8-4ec7-b5d8-edaa958957aa/pmvs/recon0/ option-0000
returned with code 35072.
```

any help will be very helpful

**Comment by pierotofy** [comment]
Hi, thanks for reporting the issue.

It might be related to this: https://github.com/OpenDroneMap/OpenDroneMap/issues/28

Possible causes: 
 - Dataset is too large (or AWS instance doesn't have sufficient memory)
 - The images don't have sufficient overlap
 - The images are too small

Perhaps you could provide us with more details about the dataset you are trying to process?

**Comment by boxerone** [comment]
Ciao Piero
reading through that post is seems that the memory might be the problem. I am running a small instance just for testing purpose but I will definitly try with a "bigger" one.
I am processing one of the comunity dataset:
https://github.com/OpenDroneMap/odm_data_bellus/

I will try again and let you know.
thansk
adriano

**Comment by pierotofy** [comment]
If you are using a small instance, try to enable a swap file if you don't want to spend for a more memory: https://www.digitalocean.com/community/tutorials/how-to-add-swap-space-on-ubuntu-16-04

**Comment by pierotofy** [comment]
By the way, I'm also having the same error processing that dataset (my processing node uses 4 gb ram). I'm investigating right now...

**Comment by boxerone** [comment]
Correct same as mine.
Let me try increasing ram...

Il 27 Nov 2016 02:10, "Piero Toffanin" <notifications@github.com> ha
scritto:

**Comment by pierotofy** [comment]
I can confirm this is a memory issue; when I resize the images to 1800 using the "--resize-to 1800" option, processing continues without the error.

**Comment by boxerone** [comment]
Great!
I've just switched to a m4.xlarge instace and processing is running without problems.

Quite frankly it seems that now is stucked at 60%, "Generating Overview Tiles"....
![cpu](https://cloud.githubusercontent.com/assets/13165725/20669482/b8a02a9c-b574-11e6-9bf8-dbe28c16a31b.JPG)

**Comment by pierotofy** [comment]
Could be related to #55 

If you get the UUID of the task assigned by node-OpenDroneMap (look for /var/www/data/<UUID>/ in the console output), take the UUID and the URL of the node-OpenDroneMap API (for example, http://nodeodm.masseranolabs.com:80/task/UUID/download/all.zip ).

If the download starts, this is likely the same issue as #55.

Also in the output of WebODM (from `python manage.py runserver`) if you see:

```
INFO Acquiring lock: Task ID: xx
INFO Processing status: 40 for Task ID: xx
```

This is probably also the same as #55.

**Comment by boxerone** [comment]
So, processing the same images dataset, from the web interface of the node-odm, provides rusults with no problem.
Reprocessing from Web-odm, got the same error

**Comment by pierotofy** [comment]
I will investigate this and get back to you.

**Comment by pierotofy** [comment]
Sorry for the late reply! This issue should be now solved as part of the fix for #55. I'll close this for now, unless the problem continues.
